runtime/client/ext-app-entry.js

```diff
@@ -60,28 +116,38 @@ function main () {
   process.title = `${process.argv[0]} yoda-app ${target}`
   var pkg = require(`${target}/package.json`)
   logger.log(`load target: ${target}/package.json`)
+  var appId = pkg.name
+  logger = require('logger')(`entry-${appId}`)
 
   var main = `${target}/${pkg.main || 'app.js'}`
   var handle = require(main)
```

Comment:
Is it better to add error trap here?

```suggestion
  var handle
  try {
    handle = require(main)
  } catch (err) {
    // log here
  }
```

Reply:
Since this entry was running on a separated process, any error encountered here would cause process to be exited with non-zero exit code. Though a better error log would be preferred, I'd fix it.